src/ground-truth.rkt

```diff
@@ -1,10 +1,34 @@
 #lang racket
 
-(require math/bigfloat rival)
-(require "syntax/sugar.rkt" "syntax/types.rkt"
-         "common.rkt" "compiler.rkt" "sampling.rkt" "timeline.rkt"
-         "errors.rkt")
-(provide sample-points batch-prepare-points make-search-func eval-progs-real)
+(require math/bigfloat
+         rival)
+
+(require "syntax/types.rkt"
+         "common.rkt"
+         "compiler.rkt")
+
+(provide eval-progs-real
+         ground-truth-require-convergence 
```

Comment:
Exposing this is quite ugly; I suppose the abstraction boundary here has moved to `compiler.rkt`?

Reply:
I think so